docs/excel/custom-functions-runtime.md

```diff
@@ -59,6 +59,10 @@ function StoreValue(key, value) {
 }
 ```
 
+## Compare with shared runtime
+
+Need UI integration, richer Office.js objects, or advanced events? Move those functions to a [shared runtime](../testing/runtimes.md#shared-runtime) so they can use those features.
```

Comment:
I don't understand what a rich Office.js object is, or advanced events (in your custom function?)